beta/src/pages/apis/react/Suspense.md

```diff
@@ -2,17 +2,126 @@
 title: Suspense
 ---
 
-<Wip>
-
-This section is incomplete, please see the old docs for [Suspense](https://reactjs.org/docs/react-api.html#reactsuspense).
-
-</Wip>
+<Intro>
 
+`Suspense` is a React Component that displays a fallback until its children have finished loading.
 
-<Intro>
 
 ```js
-<React.Suspense fallback={<Spinner />}>
+<Suspense fallback={<Placeholder />}>
+  <ComponentThatLoadsData />
+</Suspense>
 ```
 
 </Intro>
+
+- [Usage](#usage)
+  - [Displaying a fallback while data is loading](#displaying-a-fallback-while-data-is-loading)
+  - [Revealing nested content as it loads](#revealing-nested-content-as-it-loads)
+  - [Lazy loading components with Suspense](#suspense-for-code-splitting)
+  - [Fetching data with Suspense](#suspense-for-data-fetching)
+- [Reference](#reference)
+  - [Props](#props)
+- [Troubleshooting](#troubleshooting)
+  - [How do I prevent the UI from being replaced by a fallback during an update?](#preventing-unwanted-fallbacks)
+
+---
+
+## Usage {/*usage*/}
+
+### Displaying a fallback while data is loading {/*displaying-a-fallback-while-data-is-loading*/}
+
+You can wrap any part of your application with a Suspense component. If the data in its children hasn't loaded yet, React will switch to rendering the `fallback` prop instead. For example:
+
+```js [[1, 2, "<Post />"], [2, 3, "<LoadingSpinner />"], [3, 4, "<Comments />"]]
+<>
+  <Post />
+  <Suspense fallback={<LoadingSpinner />}>
+    <Comments />
+  </Suspense>
+</>
+```
+
+Suppose that <CodeStep step={3}>`<Comments />`</CodeStep> takes longer to load than <CodeStep step={1}>`<Post />`</CodeStep>. Without a Suspense boundary, React wouldn't be able to show either component until both have loaded — <CodeStep step={1}>`<Post />`</CodeStep> would be blocked by <CodeStep step={3}>`<Comments />`</CodeStep>.
```

Comment:
I think the "code step" usage here is a bit more distracting than helpful. I'd prefer if we keep it limited and instead of highlighting pieces of JSX in code, we use it for terms like "content" and "loading indicator" and try to use at most three (ideally two) colors at the time for any given explanation. We also don't have to highlight all matches. E.g. I think it would actually be better to remove highlights here but add them in the sentence above the snippet for "its children" and "fallback prop" instead.

Ditto for sections below.